pt/lessons/basics/iex-helpers.md

```diff
@@ -1,13 +1,13 @@
 ---
-version: 1.0.1
+version: 1.0.2
 title: IEx Helpers
 ---
 
 {% include toc.html %}
 
 ## Visão Geral
 
-Assim que você começar a trabalhar com Elixir, IEx será o seu melhor amigo. Ele é um REPL, mas possui muitas funcionalidades avançadas que podem tornar a vida mais fácil, quando explorando um novo código ou desenvolvendo seu próprio trabalho a medida que avança. Há uma enorme quantidade de helpers embutidos que nós iremos detalhar nesta lição.
+Assim que você começar a trabalhar com Elixir, IEx será o seu melhor amigo. Ele é um REPL, mas possui muitas funcionalidades avançadas que podem tornar a vida mais fácil, seja explorando um novo código ou desenvolvendo seu próprio trabalho a medida que avança. Há uma enorme quantidade de helpers embutidos que nós iremos detalhar nesta lição.
```

Comment:
@doomspork The translation files shouldn't be ignored by the Typo CI? https://github.com/elixirschool/elixirschool/blob/master/.typo-ci.yml#L29 The config is correct?

Reply:
so, I tried once using the Typo CI for the `pt_BR` dictionary, so the typo with the Portuguese language has some much trouble and basic you need put the entire words in the excluded list.

Reply:
I'll have to look into Typo CI more. It seems to be noisy :|